server/utils/azureAppInsights.test.ts

```diff
@@ -26,7 +26,7 @@ const createContext = (username: string, activeCaseLoadId: string) =>
         },
       },
     },
-  } as ContextObject)
+  }) as ContextObject
```

Comment:
Change as suggested by eslint. @psoleckimoj @tpmcgowan could you please check if this is correct?

Reply:
Run it through `npm run typecheck` and it'll make sure TypeScript is happy 👍

Reply:
thanks.. no issues spotted